packages/postgres-database/tests/test_utils_migration.py

```diff
@@ -48,3 +48,6 @@ def test_migration_upgrade_downgrade(make_engine: Callable):
     assert inspector.get_table_names() == [
         "alembic_version"
     ], "Only the alembic table should remain, please check!!!"
+    # NOTE: We should remove comment
+    # assert inspector.get_enums() == []
+    # NOTE: Triggers are not implemented https://docs.sqlalchemy.org/en/20/core/reflection.html
```

Comment:
this is all commented why?

Reply:
@sanderegg Because it needs to be firstly fixed in the migrations (all the enums and triggers that are not cleaned up, then we can extend the test by these lines)

Reply:
Also these lines were not there before, I have added them there.

Reply:
well if you do not put a test that prevents what just happened, I don't see the point in adding comments like these

Reply:
i would also remove the commented code /ocmments if not needed.